dev/Sumangrewal/main.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import confusion_matrix,accuracy_score
+
+dataset = pd.read_csv("winequality.csv")
```

Comment:
Please load the dataset from the correct relative path in the repo so that it runs without modification, here and in the notebook.

Reply:
Yes, @dzeber I have made the changes.